frontend/public/locales/en/learn/from-work-to-retirement.json

```diff
@@ -1,5 +1,15 @@
 {
     "header": "From work to retirement",
+    "breadcrumbs": {
+        "home":{
+          "text":"Retirement Hub",
+          "link":"/home"
```

Comment:
Instead of `/home` put `/` everywhere. Middleware will be responsible to redirect the user to `/home`

Reply:
Fixed